CMakeLists.txt

```diff
@@ -39,6 +39,16 @@ elseif(CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
   FIND_PACKAGE(Threads)
 endif(CMAKE_SYSTEM_NAME MATCHES "Linux")
 
+if(WIN32)
+  set(WIN32_WINNT "0x0A00" CACHE STRING "Targeted Windows version.")
+  add_definitions(-D_WIN32_WINNT=${WIN32_WINNT})
```

Comment:
who's consuming this macro? could you define it in `src/include/config-h.in.cmake` instead?

Reply:
The Windows headers are using it (e.g. the ones from Mingw or Windows SDK), I should probably add an inline comment. Thanks for mentioning ``src/include/config-h.in.cmake``.

Reply:
I think I prefer using ``add_definitions`` instead of relying on ``acconfig.h`` in this case, making sure that the Windows includes always use the right version.

Reply:
agreed!